releases/2.13.4.md

```diff
@@ -44,12 +44,20 @@ Additionally,
 * singleton types no longer prematurely widen [#9209](https://github.com/scala/scala/pull/9209)
 * tuples are handled properly [#9147](https://github.com/scala/scala/pull/9147)/[#9163](https://github.com/scala/scala/pull/9163)/[#9147](https://github.com/scala/scala/pull/9147)
 
-If some resulting warnings are unwanted, your options include:
+Warnings reported should be resolved by:
+
+1. adding any missing cases
+2. in the case of complementary guards (e.g. `if n > 0` and `if n <= 0`) by dropping the last guard
+4. for custom extractors: demarking irrefutable extractors as such, by defining the return type as `Some`
+3. for sealed types: marking traits or parent classes sealed, parent classes abstract, and classes final
+
+Otherwise, your options for suppressing warnings include:
```

Comment:
```suggestion
New warnings reported should be resolved by:

1. adding any missing cases
2. in the case of complementary guards (e.g. `if n > 0` and `if n <= 0`) by dropping the last guard
3. for custom extractors: demarking irrefutable extractors as such, by defining the return type as `Some`
4. for sealed types: marking traits or parent classes sealed, parent classes abstract, and classes final

Otherwise, your options for suppressing warnings include:
```